x-pack/plugins/index_lifecycle_management/server/plugin.ts

```diff
@@ -22,10 +22,10 @@ import { Dependencies } from './types';
 import { registerApiRoutes } from './routes';
 import { License } from './services';
 import { IndexLifecycleManagementConfig } from './config';
-import { isEsError } from './shared_imports';
 
 const indexLifecycleDataEnricher = async (
   indicesList: IndexWithoutIlm[],
+  // TODO replace deprecated ES client after Index Management is updated
```

Comment:
Thanks for adding this `TODO`! It may also be worth adding a comment to https://github.com/elastic/kibana/issues/73973 as a reminder to circle back around to this.